cmake/std/atdm/cee-rhel6/environment.sh

```diff
@@ -0,0 +1,113 @@
+################################################################################
+#
+# Set up env on a CEE RHEL6 system for ATMD builds of Trilinos
+#
+# This source script gets the settings from the JOB_NAME var.
+#
+################################################################################
+
+if [ "$ATDM_CONFIG_COMPILER" == "DEFAULT" ] ; then
+  export ATDM_CONFIG_COMPILER=GNU
+fi
+
+if [ "$ATDM_CONFIG_KOKKOS_ARCH" == "DEFAULT" ] ; then
+  unset ATDM_CONFIG_KOKKOS_ARCH
+else
+  echo
+  echo "***"
+  echo "*** ERROR: Specifying KOKKOS_ARCH is not supported on RHEL6 ATDM builds"
```

Comment:
Hm, why not?  This means that the builds aren't exercising Kokkos' preferred CMake logic path.  Kokkos prefers the path of setting `KOKKOS_ARCH`, because it enables architecture-specific optimizations.